docs/data/material/customization/theming/theming-pt.md

```diff
@@ -223,8 +223,8 @@ This component takes a `theme` prop and applies it to the entire React tree that
 
 #### Props
 
-| Name               | Type                                     | Description                                                                                                                                                                                               |
-|:------------------ |:---------------------------------------- |:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Name             | Type                                     | Description                                                                                                                                                                                               |
+| :--------------- | :--------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
I'm not sure how these got changed. I assume Prettier did this?

Reply:
This might come from the migration when I did the doc restructuring.

Reply:
I had a look. I have tested the assumption of Jun. It's invalidated, proof, see the latest commit on HEAD: https://github.com/mui/material-ui/commit/9d6e08ba6883211828cdfcc44703ad7e2b131514.

See:

https://github.com/mui/material-ui/blob/9d6e08ba6883211828cdfcc44703ad7e2b131514/scripts/prettier.js#L64-L67

Reply:
Nice catch @oliviertassinari! It hadn't occurred to me that that could cause issues.